src/assets/v1.3.0/compatibility_fallback_handler.json

```diff
@@ -16,6 +16,8 @@
     "41": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "42": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "5": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "48": "0x0D2068Bbe4e3975adc8D6701234fBdA115CdAc19",
```

Comment:
The contracts should be deployed also in mainnet using the deterministic addresses.